spinn_front_end_common/interface/interface_functions/preallocate_resources_for_extra_monitor_support.py

```diff
@@ -38,83 +35,19 @@ def __call__(
         :rtype: ~pacman.model.resources.PreAllocatedResourceContainer
         """
 
-        progress = ProgressBar(
-            len(list(machine.ethernet_connected_chips)) + machine.n_chips,
-            "Preallocating resources for Extra Monitor support vertices")
+        progress_bar = ProgressBar(
+            1, "Preallocating resources for Extra Monitor support vertices")
```

Comment:
Is a progress bar over one item necessary?

Reply:
nope